packages/lib-classifier/src/plugins/drawingTools/components/draggable/draggable.js

```diff
@@ -121,7 +121,7 @@ function draggable(WrappedComponent) {
           <WrappedComponent
             ref={this.wrappedComponent}
             {...rest}
-            dragging={dragging}
+            dragging={dragging.toString()}
```

Comment:
This is going to break everywhere else in the code the rely on dragging being either `true` or `false`. Te reason for that is that `'false'` is truthy in JS, so `if (dragging)` will now be true throughout the code even when we're not dragging.

This needs to be changed back and a different solution found.

Reply:
Yes I was worried it would break many things as well! I quickly tried the single image viewer (I Fancy Cats) and the LCV (TESS) and dragging mode and annotation mode appeared to work as expected. Is there a specific viewer you think would most like break or user actions I'm missing?

Reply:
I think the solution we need is that the `Draggable` HOC should use a different prop name, or the `dragging` React prop should not be passed down to HTML elements as an element attribute.

Reply:
Ah yes I see a new warning as expected with these changes - "Warning: Failed prop type: Invalid prop `dragging` of type `string` supplied to `ForwardRef(observerComponent)`, expected `boolean`.)" when dragging marks.

Reply:
@mcbouslog it will probably break drawing marks styling here:
https://github.com/zooniverse/front-end-monorepo/blob/103fc15dd0428a843d7c1a85a93b4951c9edb67e/packages/lib-classifier/src/plugins/drawingTools/components/Mark/Mark.js#L20-L26

I did a quick GitHub searching for `dragging`, and it looks like drag handles use it too. Maybe individual drawing tools also.